README.md

```diff
@@ -43,14 +48,47 @@ bash run_pipeline.sh
 
 For exact instructions on how to reproduce our analysis see `run_pipeline.sh`.
 
+## Results
+
+Our _in silico_ deconvolution of CD4+ cells, CD8+ cells, and Macrophages in TCGA
+data matches very closely to immunohistochemistry estimates of the same cell types
+in a separate dataset. The proportions of immune cell infiltrate across subtypes
+corresponds strongly.
+
+![immune deconvolution and IHC](figures/boxplot_validation_TCGA_summary.png?raw=true)
+
+We also observed that high macrophage infiltration was associated with worse outcomes
+in the TCGA dataset. This relationship was strengthed after adjusting for several
+covariates including age, gender, and gene expression based subtype.
+
+![TCGA macrophage survival](figures/TCGA_kaplanmeier_Macrophages.png?raw=true)  
+
+
 ## Contact
 
 For all code related questions, bug reporting, or feature requests please file a
 [GitHub issue](https://github.com/greenelab/GBM_immune_profiles/issues "file an issue")
 
 ## Dependencies
 
-All code was performed in R version 3.2.3 and packages were managed by the
-[checkpoint](https://cran.r-project.org/web/packages/checkpoint/index.html "Microsoft checkpoint r package")
-package set at "2016-08-16" (see `install.R` for more details)
+All analyses were performed in R version 3.2.3 and packages were versioned with the
+checkpoint package (version 0.3.18) set to a snapshot date of "2016-08-16". The checkpoint
+package will automatically download all the specified packages at the versions they
+existed in on that specific date. See [install.R](install.R) for more details.
+The versions for each package are:
+
+| Package | Version |
```

Comment:
Not a huge fan of this table... since it isn't guaranteed to reflect the reality.

How about at the end of your script you run `sessionInfo()` and save the output to a file that gets committed back to GitHub? Then you can remove all version info from the README.